alvr/common/src/connection_result.rs

```diff
@@ -65,7 +65,10 @@ pub trait HandleTryAgain<T> {
 impl<T> HandleTryAgain<T> for io::Result<T> {
     fn handle_try_again(self) -> ConResult<T> {
         self.map_err(|e| {
-            if e.kind() == io::ErrorKind::TimedOut || e.kind() == io::ErrorKind::WouldBlock {
+            if e.kind() == io::ErrorKind::TimedOut
+                || e.kind() == io::ErrorKind::WouldBlock
+                || e.raw_os_error() == Some(997)
```

Comment:
Should probably do this for windows only since error codes on linux differ completely

Reply:
Uh good call